unit-tests/live/options/test-options-watcher.py

```diff
@@ -0,0 +1,52 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:device each(D400*)
```

Comment:
Really needed for each camera?

Reply:
No, just checking the mechanism. 
Removed `each` and added `device D500*` for configurations that might not have D400 devices attached